test/index.js

```diff
@@ -545,6 +545,28 @@ describe('def', function() {
                    'as its first argument; received ["abc"]'));
   });
 
+  it('supports "enum" types', function() {
+    var def = $.create($.env.concat([$.Enum, $.ValidDate, $.ValidNumber]));
```

Comment:
I've prepended `TimeUnit` with a `$` to make it consistent with the convention you followed in plaid/sanctuary/pull/123

:zap: